charts/prefect-agent/templates/deployment.yaml

```
@@ -48,6 +48,9 @@ spec:
       {{- if .Values.agent.podSecurityContext }}
       securityContext: {{- .Values.agent.podSecurityContext | toYaml | nindent 8 }}
       {{- end }}
+      {{- if .Values.server.priorityClassName }}
```

This looks like a typo

```suggestion
      {{- if .Values.agent.priorityClassName }}
```